src/js/containers/covid19/Covid19Container.jsx

```diff
@@ -64,7 +64,7 @@ const Covid19Container = () => {
                 defCodesRequest.current = fetchDEFCodes();
                 try {
                     const { data } = await defCodesRequest.current.promise;
-                    dispatch(setDEFCodes(data.codes.filter((c) => c.disaster === 'covid_19')));
+                    dispatch(setDEFCodes(data.codes));
```

Comment:
Same as above. This should not be the case. Defcodes in the COVID 19 filter in Redux should be COVID Defcodes.